### PR TITLE
TEST/IODEMO: Fix server params in run_iodemo script

### PR DESCRIPTION
**Pull request by brminich**
## What
Propagate all relevant parameters to server in run_iodemo script. The following params are added:
-k <chunk-size> 
-r <io-request-size> 
-b <number of buffers> ( Number of offcache IO buffers)
-v                         (Set verbose mode)
-H                         (Use human-readable timestamps)

### CODE REVIEW

**Review by dmitrygx — APPROVED** [review]

**Comment by dmitrygx** [comment]
double kill :)
#5913
#5840

**Comment by brminich** [comment]
@yosefe, please review

File: test/apps/iodemo/run_io_demo.sh [hunk 1]
```diff
@@ -299,12 +299,33 @@ build_server_args_list() {
...
+		-r)
+			value="$2"
+			iodemo_server_args+=" $key $value"
+			shift
+			;;
+		-b)
+			value="$2"
+			iodemo_server_args+=" $key $value"
+			shift
+			;;
```
Comment (yosefe) [inline test/apps/iodemo/run_io_demo.sh:316]: now there are many params
i think we can unite args like this:
```
-d|-P|-k| ... )
			value="$2"
			iodemo_server_args+=" $key $value"
			shift
			;;
-q|-v|...)
			iodemo_server_args+=" $key"
			;;
```

Comment (brminich) [reply test/apps/iodemo/run_io_demo.sh]: fixed

**Review by yosefe — APPROVED** [review]
pls squash

**Comment by brminich** [comment]
@yosefe, ok to merge?

**Pull request merged**